cmd/kusk/cmd/deploy.go

```diff
@@ -107,6 +107,7 @@ var deployCmd = &cobra.Command{
 				if err := k8sclient.Update(ctx, api, &client.UpdateOptions{}); err != nil {
 					return err
 				}
+				fmt.Println("Updated", api.Name, "API")
```

Comment:
```suggestion
			fmt.Println("api.gateway.kusk.io/%s updated\n", api.Name)
```

Reply:
```go
fmt.Println("api.gateway.kusk.io/%s updated\n", api.Name)
```

I don't think that's correct as it will print:

```
api.gateway.kusk.io/%s updated
 oauth2-api
```

You want:

```go
fmt.Printf("api.gateway.kusk.io/%s updated\n", api)
```